modules/rosa-sdpolicy-instance-types.adoc

```diff
@@ -18,10 +18,24 @@ endif::rosa-with-hcp[]
 ifndef::rosa-with-hcp[]
 Single availability zone clusters require a minimum of 3 control plane nodes, 2 infrastructure nodes, and 2 worker nodes deployed to a single availability zone.
 
-Multiple availability zone clusters require a minimum of 3 control plane nodes, 3 infrastructure nodes, and 3 worker nodes. Additional nodes must be purchased in multiples of three to maintain proper node distribution.
+Multiple availability zone clusters require a minimum of 3 control plane nodes, 3 infrastructure nodes, and 3 worker nodes.
 
+Consider the following limitations when deploying and managing workloads:
 
-Control plane and infrastructure nodes are deployed and managed by Red{nbsp}Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the Red{nbsp}Hat Operator support section below for more information about Red{nbsp}Hat workloads that must be deployed on worker nodes.
+* You must deploy workloads on worker nodes that exist in the cluster by using {product-title} machine pools.
+* Run only node-level workloads on the control plane and infrastructure nodes.
+* Ensure that any workloads running on these nodes are secure, scalable, and compatible with a version of {product-title}, so that Service Level Agreements (SLAs) for API server availability is not impacted.
```

Comment:
s/Agreements/Agreement (SLA)

Also, can we make this line more unambiguous to say that this "responsibility" rests with the customer?